### DISCUSSION TITLE
Problem with Anaconda, oh-my-posh and Git Bash for Windows

### DISCUSSION THREAD

**Discussion by afigueroa1xp** [JanDeDobbeleer/oh-my-posh#2975] [Troubleshoot] [answered]
Hello

I'm having a problem with oh-my-posh and Anaconda using Git bash for Windows... Everytime I want to install a package from Anaconda in my Python environment, I get this error:

`# >>>>>>>>>>>>>>>>>>>>>> ERROR REPORT <<<<<<<<<<<<<<<<<<<<<<`

    Traceback (most recent call last):
      File "C:\Users\pc1\anaconda3\lib\site-packages\conda\exceptions.py", line 1129, in __call__
        return func(*args, **kwargs)
      File "C:\Users\pc1\anaconda3\lib\site-packages\conda\cli\main.py", line 111, in main_sourced
        print(activator.execute(), end="")
      File "C:\Users\pc1\anaconda3\lib\encodings\cp1252.py", line 19, in encode
        return codecs.charmap_encode(input,self.errors,encoding_table)[0]
    UnicodeEncodeError: 'charmap' codec can't encode character '\ue200' in position 98: character maps to <undefined>

`$ C:\Users\pc1\anaconda3\Scripts\conda-script.py shell.posix reactivate`
`An unexpected error has occurred. Conda has prepared the above report.`

This issue doesn't occur in PowerShell, only in Git Bash… When I deactivate oh-my-posh in Git Bash, the issue disappear. What could be the reason of this error?

**Comment by JanDeDobbeleer** [comment, marked as answer]
@afigueroa1973 this is a UTF-8 error and thus a Conda bug. Python3 should come with UTF-8 out of the box, 2 doesn't. However your shell could be not set to UTF-8 (see 1252 encoding) and Conda could be trying to read the prompt. Theoretically [changing Python (or your shell) to use UTF-8](https://gankrin.org/how-to-enable-utf-8-in-python/) could change this, nonetheless it's a Conda bug.

**Reply by afigueroa1xp** [reply]
Thank you for your quick answer…

After research about UTF-8 problems on Python and Windows, I found that this problem is resolved creating a Python UTF-8 environment variable on Windows

`PYTHONUTF8=1`

That, solved my problem… Thanks again for the tip.
